platform/security/cas/security-cas-client/pom.xml

```diff
@@ -166,22 +168,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.44</minimum>
+                                            <minimum>0.48</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.21</minimum>
+                                            <minimum>0.31</minimum>
                                         </limit>
                                         <limit>
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.24</minimum>
+                                            <minimum>0.30</minimum>
                                         </limit>
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.47</minimum>
+                                            <minimum>0.46</minimum>
```

Comment:
Well the "actual" values (before my changes) where actually : .49, .26, .29, .49 respectively... So actually Instruction and line went down while branch and complexity went up...This is because I removed the TestPage and the TestPageTest and the test must have covered a "better than average" amount of the TestPage but the complexity and branches of the class must have been "lower than average". I figured I should update all of them since they are all below the defaults.